verifier/src/lib.rs

```diff
@@ -161,6 +161,7 @@ impl CertVerifier for P384CertVerifier {
 }
 
 /// This struct encapsulates the signature verification process for a PkiPath.
+#[derive(Debug)]
```

Comment:
Was this intentional? If it's useful to you I'd like to keep the change but if it's something more akin to left over debug code it should probably be removed.

Reply:
It's required to implement [`rustls::ServerCertVerifier`](https://docs.rs/rustls/latest/rustls/client/danger/trait.ServerCertVerifier.html)